2-WebApp-graph-user/2-1-Call-MSGraph/Controllers/HomeController.cs

```diff
@@ -49,13 +82,16 @@ public async Task<IActionResult> Profile()
                     ViewData["Photo"] = Convert.ToBase64String(photoByte);
                 }
             }
-            catch (System.Exception)
+            catch (Exception pex)
             {
+                Console.WriteLine($"{pex}");

```

Comment:
yes ... if the user has not set their photos you'll get an exception, and that's ok ... not sure if you want to write it fully. Or maybe just the message?

Reply:
Some kind of indication is good - otherwise I'd wonder "why is the app not displaying a photo?"